doc/source/georeferencing.md

```diff
@@ -0,0 +1,214 @@
+ corresponds to {attr}`geopandas.GeoSeries.total_bounds` (total bounds of all geometry features) converted to a {class}`rasterio.coords.BoundingBox`.
+
+To reproduce the behaviour of {attr}`geopandas.GeoSeries.bounds` (per-feature bounds) with a 
+{class}`~geoutils.Vector`, use {attr}`~geoutils.Vector.geom_bounds`.
+```
+
+### Footprints
+
+As reprojections between CRSs deform shapes, including extents, it is often better to consider a vectorized footprint 
+to calculate intersections in different projections. The {class}`~geoutils.Raster.footprint` is a 
+{class}`~geoutils.Vector` object with a single polygon geometry for which points have been densified, allowing 
+reliable computation of extents between CRSs.
+
+```{code-cell} ipython3
+# Print raster footprint
+rast.get_footprint_projected(rast.crs)
+# Plot vector footprint
+vect.get_footprint_projected(vect.crs).plot()
+```
+
+### Grid (only for rasters)
+
+A raster's grid origin and resolution are defined by its geotransform attribute, {attr}`~geoutils.Raster.transform`, and its shape by the data array shape.
+From it are derived the resolution {attr}`~geoutils.Raster.res`, the 2D raster shape 
+{attr}`~geoutils.Raster.shape` made up of its {attr}`~geoutils.Raster.height` and {attr}`~geoutils.Raster.width`.
+
+```{code-cell} ipython3
+# Get raster transform and shape
+print(rast.transform)
+print(rast.shape)
+```
+
+(pixel-interpretation)=
+### Pixel interpretation (only for rasters)
+
+A largely overlooked aspect of a raster's georeferencing is the pixel interpretation stored in the 
+[AREA_OR_POINT metadata](https://gdal.org/user/raster_data_model.html#metadata). 
+Pixels can be interpreted either as **"Area"** (the most common) where **the value represents a sampling over the region 
+of the pixel (and typically refers to the upper-left corner coordinate)**, or as **"Point"**
+where **the value relates to a point sample (and typically refers to the center of the pixel)**, used often for DEMs. 
+
+Pixel interpretation is stored as a string in the {attr}`geoutils.Raster.area_or_point` attribute. 
+
+```{code-cell} ipython3
+# Get pixel interpretation of raster
+rast.area_or_point
+```
+
+Although this interpretation is not intended to influence georeferencing, it **can influence sub-pixel coordinate
+interpretation during analysis**, especially for raster–vector–point interfacing operations such as point interpolation, 
+or re-gridding, and might also be a problem if defined differently when comparing two rasters.
+
+```{important}
+By default, **pixel interpretation can induce a half-pixel shift during raster–point interfacing** 
+(mirroring [GDAL's default ground-control point behaviour](https://trac.osgeo.org/gdal/wiki/rfc33_gtiff_pixelispoint)), 
+but only **raises a warning for raster–raster operations**.
+
+This behaviour can be modified at the package-level by using [GeoUtils' configuration parameters]() 
```

Comment:
Because the link is empty, when clicking on it, it points to the same page again. Something to be added later I guess?

Reply:
Added!